### PR TITLE
Setup most watched page furniture

### LINKED ISSUE #7640: Setup most watched page with page furniture and translated h1 page title
Example map: [Miro](https://miro.com/app/board/o9J_kvZ2wMk=/?moveToWidget=3074457349544789098&cot=12)
Some analysis of existing routes: [Dropbox Paper](https://paper.dropbox.com/doc/Analysis-Most-Watched--A6ccw62pyGe4I0pgfBjDw8iDAg-TK3nz2VKkDvUNC8P1rnh7) 
Jira ticket: [Jira](https://jira.dev.bbc.co.uk/browse/WSMEDIA-234)
Design: [Zeplin](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f22de67d3d1fd1f74e4cda1)

**Is your feature request related to a problem? Please describe.**
Currently when you visit https://www.bbc.com/afaanoromoo/media/video this page is being served by PAL. This page is to be served by Simorgh and use the MostWatched component.

**Describe the solution you'd like**
When I visit https://www.bbc.com/afaanoromoo/media/video I should see a page with page furniture (header/nav etc) and the Most Watched title (translated for the service).

- set up a new route in https://github.com/bbc/simorgh/blob/latest/src/app/routes/index.js
- set up regex to match route here https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/regex/index.js
- set up a new page component with page furniture here https://github.com/bbc/simorgh/tree/latest/src/app/pages (must use loadable)
- Add new most watched page type to scripts/bundleSize/pages.js to report the bundle size after each build of simorgh
- h1 page title should be rendered on page, it comes from service config e.g. https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afrique.js#L209

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #7640

**Overall change:**
Setup most watched page with page furniture and translated h1 page title

**Code changes:**

-set up a new route in https://github.com/bbc/simorgh/blob/latest/src/app/routes/index.js
- set up regex to match route here https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/regex/index.js
- set up a new page component with page furniture here https://github.com/bbc/simorgh/tree/latest/src/app/pages (must use loadable)
- Add new most watched page type to scripts/bundleSize/pages.js to report the bundle size after each build of simorgh
- h1 page title should be rendered on page, it comes from service config e.g. https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/afrique.js#L209

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/routes/utils/regex/index.js [hunk 1]
```diff
@@ -55,6 +56,7 @@ export const mostReadPagePath = getMostReadPageRegex(allServices);
 export const mostReadDataRegexPath = getMostReadDataRegex(allServices);
 
 export const mostWatchedDataPath = getMostWatchedDataRegex(allServices);
+export const mostWatchedPagePath = getMostWatchedPageRegex(allServices);
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/index.js:60]: There might be a failing test here - snapshot tests are autogenerated for regex, so might need to be updated

File: src/app/pages/MostWatchedPage/MostWatchedPage.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  return (
+    <>
+      <ComscoreAnalytics />
+      <MetadataContainer
+        title={header}
+        lang={lang}
+        description={`${header} - ${brandName}`}
+        openGraphType="website"
+      />
+      <LinkedData type="WebPage" seoTitle={header} />
```
Comment (karinathomasbbc) [inline src/app/pages/MostWatchedPage/MostWatchedPage.jsx]: Should we be adding these components in this PR? I believe we have additional issues for adding analytics, SEO etc

File: src/app/pages/index.js [hunk 3]
```diff
@@ -10,6 +10,8 @@ export const MediaAssetPage = loadable(() => import('./MediaAssetPage'));
 
 export const MostReadPage = loadable(() => import('./MostReadPage'));
 
+export const MostWatchedPage = loadable(() => import('./MostWatchedPage'));
```
Comment (karinathomasbbc) [inline src/app/pages/index.js:13]: I believe there is also a failing bundle test, now that we have added the MostWatchedPage

File: src/app/pages/MostWatchedPage/index.test.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import MostWatchedPage from './MostWatchedPage';
+import pidginMostWatchedData from '#data/pidgin/mostWatched';
+import * as analyticsUtils from '#lib/analyticsUtils';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+
+fetch.mockResponse(JSON.stringify(pidginMostWatchedData));
+
+analyticsUtils.getAtUserId = jest.fn();
```
Comment (jroebu14) [inline src/app/pages/MostWatchedPage/index.test.jsx]: what does this do?

**Review by jroebu14 — COMMENTED** [review]
if you're tackling the analytics and metadata as well can you put those tickets into in progress?

**Review by jroebu14 — COMMENTED** [review]
Is the data fetching done as part of this PR as well? Should I move that into in progress @pharingee ?

File: src/app/routes/index.test.jsx [hunk 5]
```diff
@@ -211,6 +212,27 @@ it('should route to and render a front page', async () => {
...
+    service: 'pidgin',
+    pageType,
+  });
+  const { getByText } = renderRouter({
+    pathname,
+    pageData,
+    pageType,
+    service: 'pidgin',
+  });
+  screen.debug();
```
Comment (rebeccamcginn) [inline src/app/routes/index.test.jsx]: Needs removed

**Review by rebeccamcginn — APPROVED** [review]
Looks good locally

File: src/app/pages/MostWatchedPage/index.test.jsx (hunk 4 above)
Comment (pharingee) [reply src/app/pages/MostWatchedPage/index.test.jsx]: It helps with analytics  mocking but is not necessary. Taken out

File: src/app/pages/MostWatchedPage/MostWatchedPage.jsx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  const {
+    dir,
+    mostWatched: { header },
+  } = useContext(ServiceContext);
+
+  const MostWatchedWrapper = ({ children }) => (
+    <>
+      <IndexHeading id="content">{header}</IndexHeading>
+      <MarginWrapper>
+        <GelPageGrid
```
Comment (jroebu14) [inline src/app/pages/MostWatchedPage/MostWatchedPage.jsx:45]: We have this in live radio and OD radio pages too, don't we? What do we think about doing something like this?

https://github.com/bbc/simorgh/blob/media-team-collab-week/src/app/containers/PageHandlers/withStickyPlayer/Container.jsx

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**